diff --git a/android/src/main/java/com/revenuecat/purchases/react/ui/PaywallFooterViewManager.kt b/android/src/main/java/com/revenuecat/purchases/react/ui/PaywallFooterViewManager.kt
index e7958b1fe216ddc780a59bd414627f23a8f2268a..611a85311864f687507e48009b5b4208c2564d04 100644
--- a/android/src/main/java/com/revenuecat/purchases/react/ui/PaywallFooterViewManager.kt
+++ b/android/src/main/java/com/revenuecat/purchases/react/ui/PaywallFooterViewManager.kt
@@ -21,7 +21,7 @@ internal class PaywallFooterViewManager : BasePaywallViewManager<PaywallFooterVi
             // https://github.com/facebook/react-native/issues/17968#issuecomment-1672111483
             override fun requestLayout() {
                 super.requestLayout()
-                post(measureAndLayout)
+            //    post(measureAndLayout)
             }
 
             private val measureAndLayout = Runnable {
